envier/env.py

```diff
@@ -190,6 +199,37 @@ def __call__(self, env):
         return value
 
 
+ConfigSourceType = Literal["default", "environment", "programmatic"]
```

Comment:
Can't we make this an `Enum`?

Reply:
Oh, I opted for not since we need to support older Pythons still, is this a blocker @P403n1x87?

Reply:
🤔 I think I'm getting a bit confused about the aim again. If this is for something that needs to go in ddtrace, it would probably be a new feature, hence it will be first released in a 2.x release. There, we no longer have support for older Pythons. We also don't plan any new 1.x releases. Are we planning to use envier in other projects that still support Python 2?